Lib/test/test_mmap.py

```diff
@@ -1058,6 +1060,90 @@ def __exit__(self, exc_type, exc_value, traceback):
                 with self.assertRaisesRegex(ValueError, "mmap closed or invalid"):
                     m.write_byte(X())
 
+    @unittest.skipUnless(os.name == 'nt', 'requires Windows')
+    @unittest.skipUnless(hasattr(mmap.mmap, '_protect'), 'test needs debug build')
+    def test_access_violations(self):
+
+        code = textwrap.dedent("""
+            from test.support.os_helper import TESTFN
+            import faulthandler
+            import mmap
+            import os
+            from contextlib import suppress
+
+            faulthandler.disable()
```

Comment:
```suggestion
            # Prevent logging access violations to stderr.
            faulthandler.disable()
```
It's helpful to explain why this step is required.